doc/devguide/index.rst

```diff
@@ -0,0 +1,153 @@
+DAGMC Developers Guide
+==========
+DAGMC is an open source project that facilitates the running of CAD based particle transport problems.
+
+Contributing
+~~~~~~~~~~~~
+Contributing to the DAGMC project is very straightforward, DAGMC is hosted on Github where issues and pull requests are 
```

Comment:
Are you sure about this one? The full sentence as it stands right now is "Contributing to the DAGMC project is very straightforward, DAGMC is hosted on Github where issues and pull requests are discussed and merged."